templates/patterns/button/pattern-button.html.twig

```diff
@@ -4,11 +4,35 @@
  * Button.
  */
 #}
-{% include '@ecl/ec-component-button' with {
-  'modifier': variant|replace({'_':'-'}),
-  'type_attribute': type_attribute,
-  'href': href,
-  'icon': icon,
-  'label': label,
-  'is_block': is_block
-} only %}
+{% if variant == 'default' %}
+  {% include '@ecl/link' with {
+    'link': {
+      'type': type_attribute,
+      'label': label,
+      'path': href,
+      'icon_position': icon_position|default('after')
+    },
+    'icon': {
+      'path': '/dist/images/icons/sprites/icons.svg',
```

Comment:
Let's move this into a global variable that will be always available in all templates, we can achieve that by implementing `function hook_preprocess(&$variables, $hook)`, something like:

```php
function oe_theme_preprocess(&$variables, $hook) {
  global $base_url;
  $theme = \Drupal::theme()->getActiveTheme();
  $variables['ecl_icon_path'] = $base_url.'/'. $theme->getPath() . '/dist/images/icons/sprites/icons.svg';
}
```
and then we simply do this everywhere:
```
       'path': ecl_icon_path,
```
Not that this path `'/dist/images/icons/sprites/icons.svg'` works by chance since the server root points at the root of the project in the theme, but it would not work on a site.

Reply:
done.